qiskit/circuit/bit.py

```diff
@@ -70,7 +70,7 @@ def index(self, value):
 
     def __repr__(self):
         """Return the official string representing the bit."""
-        return "%s(%s, %s)" % (self.__class__.__name__, self._register, self._index)
+        return "%s[%s]" % (self._register.name, self._index)
```

Comment:
Why is this necessary?
```suggestion
        return "%s(%s, %s)" % (self.__class__.__name__, self._register, self._index)
```

Reply:
it was for my debugging.